eventbridge-lambda-terraform/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
 	aws = {
 	  source  = "hashicorp/aws"
-	  version = "~> 4.22"
+	  version = "~> 5.0"
```

Comment:
note: Older version does not support Node.js 22 runtime. So I updated the version to v5.